ui/src/components/Tasks/DailyTasks/style.module.scss

```diff
@@ -115,7 +115,7 @@
   color: $color-input-text;
   font-size: 9px;
   height: $size;
-  margin-right: 1em;
+  margin-right: 15px;
```

Comment:
What's the reason for preferring `px` over `em`? Shouldn't these values in this instance be exactly the same?